.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -3,21 +3,31 @@ name: New release template
 about: Template for creating new releases of Durable Functions
 title: ''
 labels: ''
-assignees: cgillum
+assignees: comcmaho, amdeel, davidmrdavid, bachuv
 
 ---
 
-As part of a release, the following items need to be taken care of. For any items that are not applicable, they should be crossed out using the `~~ ~~` markdown syntax with an explanation.
-
-- [ ] Run private scale tests and ensure no regressions
-- [ ] Publish updated versions of [DurableTask.Core](https://www.nuget.org/packages/Microsoft.Azure.DurableTask.Core/) and [DurableTask.AzureStorage](https://www.nuget.org/packages/Microsoft.Azure.DurableTask.AzureStorage/) to nuget.org
+**Prep Release: (assigned to:)**
+_Due: <two-businessdays-before-release>_
 - [ ] Merge all features and fixes into the `master` branch
```

Comment:
We now use `main` as our primary branch name ✨ 

```suggestion
- [ ] Merge all features and fixes into the `main` branch
```

Reply:
Thank you for the reminder! It may take me some time to get past the default reflex, so please keep pointing it out if I forget!